### PR TITLE
Revert "Remove the restriction that passing Context object only applies to trusted web applications (#4177)"

### PR DESCRIPTION
**Pull request by franklu-okta**
## Description:
- **What's changed?** 
This reverts commit c4a96fe44070f029cc0b1ba05f755058cdaeeb41.
- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-615259](https://oktainc.atlassian.net/browse/OKTA-615259)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/962ebe123db7f52bb00ee06870a84e80cf571a7b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e23fdfc8-68f4-4617-bc86-9e3c8cc6652a) | 50  | 67  | 56  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e017b6a2-0969-4a31-babf-8aba4b90e074?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 1]
```diff
@@ -87,7 +87,9 @@ The authentication transaction [state machine](#transaction-state) can be modifi
```
Comment (franklu-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:87]: This PR is to revert the change under https://github.com/okta/okta-developer-docs/pull/4177.
Per discussion here: https://okta.slack.com/archives/CP4D95VNY/p1686158762481039?thread_ts=1677695447.494849&cid=CP4D95VNY

**Review by vincentle-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 2]
```diff
@@ -87,7 +87,9 @@ The authentication transaction [state machine](#transaction-state) can be modifi
 
 ##### Context object
 
-The context object allows applications, such as an external portal, to pass additional context for the authentication or recovery transaction.
+The context object allows [trusted web applications](#trusted-application) such as an external portal to pass additional context for the authentication or recovery transaction.
+
+> **Note:** Overriding context such as `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication or recovery requests with a valid *administrator API token*. If an API token is not provided, the `deviceToken` will be ignored.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:92]:
```suggestion
> **Note:** Overriding context such as `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication or recovery requests with a valid administrator API token. If an API token is not provided, the `deviceToken` is ignored.
```
Italics isn't used in our style guide

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 3]
```diff
@@ -566,6 +568,7 @@ Authenticates a user through a [trusted application](#trusted-application) or pr
 
 **Notes:**
 
+* Specifying your own `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication requests with a valid *API token*. If an API token is not provided, the `deviceToken` will be ignored.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:571]:
```suggestion
* Specifying your own `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication requests with a valid API token. If an API token is not provided, the `deviceToken` is ignored.
```

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 4]
```diff
@@ -621,6 +624,7 @@ Authenticates a user through a [trusted application](#trusted-application) or pr
 
 **Notes:**
 
+* Specifying your own `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication requests with a valid *API token*. If an API token is not provided, the `deviceToken` is ignored.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:627]:
```suggestion
* Specifying your own `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication requests with a valid API token. If an API token is not provided, the `deviceToken` is ignored.
```

**Review by vanngo-okta — APPROVED** [review]
Just a few style nits.
Approving to not hold you back! 🚢

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/505e9b059ef461ae3f2ba1b3932b8b4bac20ccc3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2fe61cd1-1def-48ac-a6e9-f7be7cbc1b9a) | 50  | 67  | 57  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.96260241-735f-4fba-8501-9662bd5c8c34?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/c8a91dfecd962c22d1c1c6464cb9af1ea176ec61/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/95fdea61-dc18-4513-8493-4b30a2cb6306) | 50  | 67  | 57  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.03502ee2-a12a-4cb9-8915-58d332cf9aaf?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/1136c951bd63664b02dd464c4a8454fa8e32c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b407325f-5836-4952-b6b2-77a21e3b4629) | 50  | 67  | 57  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.c6d0112f-fbc6-435c-923a-bc4cdcb97889?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)
